app/javascript/courses/curricula/components/CoursesCurriculum.re

```diff
@@ -569,17 +569,24 @@ let make =
               </div>
             : React.null}
          {currentLevel |> Level.isUnlocked || accessLockedLevels
-            ? targetGroupsInLevel
-              |> TargetGroup.sort
-              |> List.map(targetGroup =>
-                   renderTargetGroup(
-                     targetGroup,
-                     targets,
-                     state.statusOfTargets,
-                   )
-                 )
-              |> Array.of_list
-              |> React.array
+            ? ListUtils.isEmpty(targetGroupsInLevel)
+                ? <div className="mx-auto py-10">
+                    <img className="max-w-sm mx-auto" src=levelEmptyImage />
+                    <p className="text-center font-semibold text-lg mt-1">
+                      {"There's no content for you to see yet." |> str}
```

Comment:
The image that is used should be reconsidered. I don't really see any connection between the image and this message.

Personally, when I see the image, I see: _"someone holding a blank paper, and asking the viewer to give them something"_.

At least, that's what I _feel_ when I look at the image of a person with their palm upturned & towards me.

I think it would have been more appropriate for this message to be: _"This level appears to be empty; the authors are probably hard at work creating new content."_

Reply:
Also, could we add a spec to check the presence of this message?